docs/HowToDebugCrashWithAddr2lineTool.md

```diff
@@ -0,0 +1,116 @@
+# How to use Addr2Line tool for debugging
+**addr2line** is a tool which comes with the Toolchain in Linux.
+It converts the given address into file names and line numbers.  
+Developers can debug the TizenRT ELF/AXF output file through **addr2line**.
+It is escpecially helpful which debugging a crash and can be used to find the 
+exact line number where the crash occured and also the function callstack
+which resulted in the crash.
+
+## Contents
+- [Prerequisites](#prerequisites)  
+- [How to use addr2line](#how-to-use-addr2line)
+- [How to debug a crash with addr2line](#how-to-debug-a-crash-with-addr2line)
+
+## Prerequisites
+To use addr2line, please enable below config.  
+
+In menuconfig, enable **CONFIG_DEBUG_SYMBOLS**.
+
+```
+menuconfig -> Debug Options -> Generate Debug Symbols
+```
+
+Also, it is easy to invoke addr2line if the the toolchain has been included in the
+system path using the PATH environment variable.
+For example, if your ARM toolchain is located at :
+
+```
+/home/tools/gcc-arm-none-eabi-9-2019-q4-major
+```
+Then, your PATH variable must include the toolchain bin folder as follows :
+
+```
+/home/tools/gcc-arm-none-eabi-9-2019-q4-major/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+```
+
+## How to use addr2line
+After build, use **addr2line** from linux shell.  
+Please refer to linux man page.  
+(https://man7.org/linux/man-pages/man1/addr2line.1.html)  
+Simple command is like below.  
+```
+$ addr2line -e [filepath] -a [address]
+```
+
+However, if the path to your toolchain is not configured as mentioned above, or if there are multiple toolchains installed on your system,
+then you might have to speficy the correct version of the addr2line to be used. It can be done as follows:
+
+Below command specifies that ARM toolchain is to be used.
```

Comment:
toolchain to be used* OR
toolchain that is to be used*

Reply:
The current statement is correct.